### PR TITLE
Mock ca certificates package

### PR DESCRIPTION
**Pull request by aloncarmel111**
What does this PR do?
Currently we are copying the ca-certificates package but not mocking this package
This  PR Mock ca-certificates package and hold the package. 

#### Change log
- [ ] The CHANGELOG has been updated, or
- [x] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [x] The changes in this PR do not require tests

#### Documentation
- [x] This PR does not require updating any documentation, or
- [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs

### CODE REVIEW

File: openssl-builder/Dockerfile [hunk 1]
```diff
@@ -46,4 +46,6 @@ COPY openssl ./
 RUN equivs-build openssl
 COPY libssl ./
 RUN equivs-build libssl
-RUN apt-get install ca-certificates
+COPY ca-certificates ./
+RUN equivs-build ca-certificates
+RUN apt-get install -y ca-certificates
```
Comment (sashaCher) [inline openssl-builder/Dockerfile:51]: Is it needed step on the builder image?

File: phusion-ruby-fips/Dockerfile [hunk 2]
```diff
@@ -29,15 +29,21 @@ RUN rm -rf /usr/local/ssl/certs
...
 RUN dpkg -i openssl_1.0.2u_all.deb
 RUN dpkg -i libssl1.0.0_1.0.2u_all.deb
+
 ## Prevent openssl and libssl package from being automatically installed, upgraded or removed
+RUN apt-mark hold ca-certificates
 RUN apt-mark hold openssl
 RUN apt-mark hold libssl1.0.0
 RUN apt-mark hold libssl1.1
 
+
```
Comment (sashaCher) [inline phusion-ruby-fips/Dockerfile:46]: Redundant

File: ubuntu-ruby-fips/Dockerfile [hunk 3]
```diff
@@ -29,15 +29,21 @@ RUN rm -rf /usr/local/ssl/certs
...
 RUN dpkg -i openssl_1.0.2u_all.deb
 RUN dpkg -i libssl1.0.0_1.0.2u_all.deb
+
 ## Prevent openssl and libssl package from being automatically installed, upgraded or remove
+RUN apt-mark hold ca-certificates
 RUN apt-mark hold openssl
 RUN apt-mark hold libssl1.0.0
 RUN apt-mark hold libssl1.1
 
+
```
Comment (sashaCher) [inline ubuntu-ruby-fips/Dockerfile:46]: Redundant

File: openssl-builder/Dockerfile (hunk 1 above)
Comment (aloncarmel111) [reply openssl-builder/Dockerfile:51]: yes we are copying this package artifacts to `phusion-ruby-fips` and `ubuntu-ruby-fips `images

File: phusion-ruby-fips/Dockerfile (hunk 2 above)
Comment (aloncarmel111) [reply phusion-ruby-fips/Dockerfile:46]: Done

File: ubuntu-ruby-fips/Dockerfile (hunk 3 above)
Comment (aloncarmel111) [reply ubuntu-ruby-fips/Dockerfile:46]: Done

**Comment by aloncarmel111** [comment]
@sashaCher Tested appliance Conjur OSS and Conjur-ui

**Pull request merged**